### PR TITLE
Use <form> to download cases in UI

### PR DESCRIPTION
**Pull request by allysonjp715**
This prevents the browser from storing the downloaded data in memory and greatly improves speed and performance:

For 9000 cases
Before change: 34.2 kB over 4.42s
Screen Shot 2020-09-21 at 4 26 07 PM

After change: 295B over 1.46s
Screen Shot 2020-09-21 at 4 28 48 PM

#1107 #1109 #1110

I think these bugs can be closed out after this PR but I want to double check in dev/prod with larger datasets.

### CODE REVIEW

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 1]
```diff
@@ -243,41 +241,35 @@ function RowMenu(props: {
...
+            >
+                <input name="query" value={props.search.trim()} />
+            </form>
+            <Button
+                variant="outlined"
+                color="primary"
+                onClick={(): void => formRef.current.submit()}
+                startIcon={<SaveAltIcon />}
+            >
+                Download
```
Comment (attwad) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:263]: we can't get the loading spinner anymore?

**Review by attwad — APPROVED** [review]
one less dependency and faster? yesssss

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/LinelistTable.tsx:263]: It's not needed anymore as the browser download object shows up quickly (the bottom panel in chrome) and shows the loading there.

Comment (attwad) [reply verification/curator-service/ui/src/components/LinelistTable.tsx:263]: nice! makes sense.

**Comment by allysonjp715** [comment]
Unfortunately had to remove the cypress tests now that the HTTP response no longer contains the data. Cypress does not support interacting with file downloads: https://github.com/cypress-io/cypress/issues/5651

**Pull request merged**